src/prommis/uky/uky_flowsheet.py

```diff
@@ -862,5 +874,719 @@ def display_results(m):
+        * flowsheet.fs.leach.liquid[0, 1].dens_mol,
+        to_units=units.ton / units.hr,
+    )
+    # Feed grade is grams of solute per 1000 grams of solution (solution ~ water since dilute concentrations)
+    feed_grade = units.convert(
+        1
+        * units.L
+        * (
+            flowsheet.fs.leach_liquid_feed.conc_mass_comp[0, "H"]
+            + flowsheet.fs.leach_liquid_feed.conc_mass_comp[0, "HSO4"]
+            + flowsheet.fs.leach_liquid_feed.conc_mass_comp[0, "SO4"]
+        ),
+        to_units=units.g,
+    )
+    m.fs.feed_input = Var(initialize=feed_input, units=units.ton / units.hr)
+    m.fs.feed_grade = Var(initialize=value(feed_grade * 1000), units=units.ppm)
+
+    hours_per_shift = 8
+    shifts_per_day = 3
+    operating_days_per_year = 336
+
+    # for convenience
+    m.fs.annual_operating_hours = Param(
+        initialize=hours_per_shift * shifts_per_day * operating_days_per_year,
+        mutable=True,
+        units=units.hours / units.a,
+    )
+
+    recovery_rate = units.convert(
+        flowsheet.fs.roaster.flow_mas_product[0], to_units=units.kg / units.hr
+    )
+    m.fs.recovery_rate_per_year = Var(
+        initialize=recovery_rate * m.fs.annual_operating_hours,
+        units=units.kg / units.yr,
+    )
+
+    # the land cost is the lease cost, or refining cost of REO produced
+    m.fs.land_cost = Expression(
+        expr=0.303736
+        * 1e-6
+        * getattr(units, "MUSD_" + CE_index_year)
+        / units.ton
+        * units.convert(m.fs.feed_input, to_units=units.ton / units.hr)
+        * hours_per_shift
+        * units.hr
+        * shifts_per_day
+        * units.day**-1
+        * operating_days_per_year
+        * units.day
+    )
+
+    solid_waste = value(
+        units.convert(
+            flowsheet.fs.leach_filter_cake.flow_mass[0], to_units=units.ton / units.hr
+        )
+    )
+
+    m.fs.solid_waste = Var(
+        m.fs.time, initialize=solid_waste, units=units.ton / units.hr
+    )  # non-hazardous solid waste
+
+    m.fs.precipitate = Var(
+        m.fs.time, initialize=0, units=units.ton / units.hr
+    )  # non-hazardous precipitate
+
+    dust = value(
+        units.convert(
+            flowsheet.fs.roaster.flow_mas_dust[0], to_units=units.ton / units.hr
+        )
+    )
+    m.fs.dust_and_volatiles = Var(
+        m.fs.time, initialize=dust, units=units.ton / units.hr
+    )  # dust and volatiles
+    m.fs.power = Var(m.fs.time, initialize=14716, units=units.hp)
+
+    resources = [
+        "nonhazardous_solid_waste",
+        "nonhazardous_precipitate_waste",
+        "dust_and_volatiles",
+        "power",
+    ]
+
+    rates = [
+        m.fs.solid_waste,
+        m.fs.precipitate,
+        m.fs.dust_and_volatiles,
+        m.fs.power,
+    ]
+
+    # define product flowrates
+    # TODO: Need to revisit this after getting costs for products defined in mw_comp_product in ree_feed_roaster
+
+    pure_product_output_rates = {
```

Comment:
@MarcusHolly if we're scaling down the inputs, we should also scale down these output rates. If the flowsheet provides these values, that would be even better.

Reply:
Some of the default sale prices for oxide products are missing - not sure if we want to resolve that now or later

Reply:
Which products are missing from the [sale price dictionary](https://github.com/prommis/prommis/blob/b903c34908f76c6dd6936a13d41baa64f45a12fd/src/prommis/uky/costing/ree_plant_capcost.py#L1680)?

Reply:
Al2O3, Ce2O3, Fe2O3, & Pr2O3

Reply:
I am not sure if we want to assign prices to Al and Fe - those are impurities we want to avoid, and thus really represent a cost penalty.

Reply:
Are Ce2O3 and Pr2O3 meaningfully different from CeO2 and Pr6O11? We do have sale prices for the latter oxides, but I assumed they weren't applicable.

Reply:
That depends on what the stable oxides of each species are. I would suspect that if we have sale prices for something it is probably the stable oxide.

Reply:
CeO2 and Pr6O11 are the stable oxides, so I'll just use the pricing for those